pkgs/applications/science/biology/deeptools/default.nix

```diff
@@ -0,0 +1,47 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, pytest
+, numpy
+, numpydoc
+, scipy
+, py2bit
+, pybigwig
+, pysam
+, matplotlib
+, plotly
+, deeptoolsintervals
+}:
+
+buildPythonPackage rec {
+
```

Comment:
```suggestion
```

Reply:
Thank you for your help on this @jonringer. 

I've removed most of the blank lines I hope. Is there any guidelines for this? I seem to always want more blank lines than necessary.

I kept the package within the biology folder. Any special reason you want to move it to science? It is a bioinformatic tool alongside samtools, bedtools etc.

Reply:
The reasoning behind moving it from python-packages, is that it's meant to be used a standalone application and not imported by other python packages. It's to avoid the situation in which someone will try to do `propagatedBuildInputs = [ deeptools ];` in another python packages, which is probably not what you want. 

Another reason for doing this, is that some applications have very specific package versions which may deviate greatly from what's available in python-modules, https://github.com/NixOS/nixpkgs/pull/71797 for example